src/NumericalAlgorithms/Spectral/SwshInterpolation.cpp

```diff
@@ -242,15 +241,17 @@ struct ClenshawRecurrenceConstants {
           ((m + gsl::at(lambda, m + static_cast<int>(l_max))) % 2) == 0 ? 1.0
                                                                         : -1.0;
       // this is the right order of the casts, other orders give the wrong
-      // answer NOLINTNEXTLINE(misc-misplaced-widening-cast)
+      // answer
+      // NOLINTNEXTLINE(bugprone-misplaced-widening-cast)
```

Comment:
[optional] here and below, one way of prevent the clang-format anarchy would be to have a blank line (or `//` empty comment line) before the `NOLINTNEXTLINE`. Just an FYI, completely up to you :)